src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
+
+In discussion about k6, some terms have a precise, technical meaning.
+If a certain term in these docs confuses you, consult this list for a definition.
 
 <Glossary>
 
+- [Application performance monitoring](#application-performance-monitoring)
+- [Concurrency](#concurrency)
 - [Correlation](#correlation)
-- [Dynamic Data](#dynamic-data)
-- [Endurance Testing](#endurance-testing)
+- [Checks](#checks)
+- [Duration](#duration)
+- [Dynamic data](#dynamic-data)
+- [Endurance testing](#endurance-testing)
+- [Executor](#executor)
 - [Goja](#goja)
-- [Horizontal Scalability](#horizontal-scalability)
-- [HTTP Archive](#http-archive)
+- [Graceful stop](#graceful-stop)
+- [Horizontal scalability](#horizontal-scalability)
+- [HTTP archive (HAR file)](#http-archive-(har-file))
+- [Iteration](#iteration)
 - [k6 Cloud](#k6-cloud)
-- [Load Test](#load-test)
+- [k6 options](#k6-options)
+- [Load test](#load-test)
+- [Load zone](#load-zone)
 - [Metric](#metric)
+- [Metric sample](#metric-sample)
 - [Parameterization](#parameterization)
-- [Performance Threshold](#performance-threshold)
 - [Reliability](#reliability)
-- [Requests per Second](#requests-per-second)
+- [Requests per second](#requests-per-second)
 - [Saturation](#saturation)
 - [Scalability](#scalability)
-- [Service-level Agreement](#service-level-agreement)
-- [Service-level Indicator](#service-level-indicator)
-- [Service-level Objective](#service-level-objective)
+- [Scenario](#scenario)
+- [Service-level agreement](#service-level-agreement)
+- [Service-level indicator](#service-level-indicator)
+- [Service-level objective](#service-level-objective)
 - [Smoke test](#smoke-test)
 - [Soak test](#soak-test)
 - [Stability](#stability)
 - [Stress test](#stress-test)
 - [System under test](#system-under-test)
-- [Test configuration](#test-configuration)
-- [Test Run](#test-run)
-- [Test Script](#test-script)
-- [User Journey](#user-journey)
-- [User Scenario](#user-scenario)
-- [Vertical Scalability](#vertical-scalability)
-- [Virtual Users](#virtual-users)
+- [Test run](#test-run)
+- [Test script](#test-script)
+- [Threshold](#threshold)
+- [Throughput](#throughput)
+- [Vertical scalability](#vertical-scalability)
+- [Virtual users](#virtual-users)
 
 </Glossary>
 
-### Correlation
-
-**Correlation** is a term used for describing the process of capturing dynamic data, received from the system under test, for reuse in subsequent requests. A common use case for correlation is retrieving and reusing a session id, or token, throughout the whole lifetime of a [virtual user](#virtual-user).
-
-### Dynamic Data
+<dl>
+ 
+<dt id="application-performance-monitoring">Application performance monitoring</dt>
+<dd>*(Or APM)*. The practice of monitoring the performance, availability, and reliability of a system. When running load tests, Grafana is an example of a compatible APM service, as users can use Grafana to monitor the load test from the side of the system under test.
```

Comment:
Consider:

The practice of monitoring the performance, availability, and reliability of a system.

k6 OSS and k6 Cloud can export testing metrics to various APM services to enable analyzing k6 metrics with your system's metrics.